kroxylicious/src/main/resources/log4j2.properties

```diff
@@ -11,7 +11,7 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} <%t> %-5p %c:%L - %m%n
 
-rootLogger.level = WARN
+rootLogger.level = INFO
```

Comment:
I'm in favour of this change but I think there is an argument that the container should probably have it's own logging configuration and possibly expose DEBUG by default.